website/docs/docs/deploy/continuous-integration.md

```diff
@@ -27,6 +27,27 @@ When the CI run completes, you can view the run status directly from within the
 
 dbt Cloud deletes the temporary schema from your <Term id="data-warehouse" /> when you close or merge the pull request. If your project has schema customization using the [generate_schema_name](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macro, dbt Cloud might not drop the temporary schema from your data warehouse. For more information, refer to [Troubleshooting](/docs/deploy/ci-jobs#troubleshooting).
 
+## Git providers who support CI
+
```

Comment:
should we add a bit more context here about what the table explains, Git providers available to what plans? e.g. 

> If your git provider has a native dbt Cloud integration, you can set up continuous integration (CI) jobs within dbt Cloud. For providers without native integration, you can still use a Git URL. The following table explains the available integration options and their capabilities.